tags/spigotmovementfix.tag

```diff
@@ -2,4 +2,4 @@ type: text
 
 ---
 
-**If you own the server, you can set `moved-wrongly-threshold` in your `spigot.yml` to the value 1.0075 to fix Bedrock movement.**
\ No newline at end of file
+If you own the server, you can set `moved-wrongly-threshold` in your `spigot.yml` to the value 1.0075 to fix Bedrock movement.
```

Comment:
Partially fix at the expense of easier hacking?

Reply:
Should be clearer now.